fastlane/lib/fastlane/setup/crashlytics_beta.rb

```diff
@@ -61,7 +66,7 @@ def lane_template
     crashlytics(api_token: '#{@beta_info.api_key}',
              build_secret: '#{@beta_info.build_secret}',#{crashlytics_path_arg}
                    emails: ['#{@beta_info.emails.join("', '")}'], # You can list more emails here
-                 # groups: ['group_alias_1', 'group_alias_2'], # You can define groups on the web and reference them here
+                 # groups: ['#{groups.join("', '")}'], # You can define groups on the web and reference them here
```

Comment:
When groups are defined by the user, we still want this to be commented out? Is that so that you don't send out the email to all testers on the first run?